routing/endpointregistry.go

```diff
@@ -38,20 +48,81 @@ var _ PostProcessor = &EndpointRegistry{}
 type RegistryOptions struct {
 }
 
+func normalizeSchemeHost(s, h string) (string, string, error) {
```

Comment:
We should stop copying this - this would be third instance I think

Reply:
It's used for NetworkBackend only, so it's more appropriate to remove it for now and think how to use it in the next PR.